config/main.py

```diff
@@ -1227,6 +1227,92 @@ def validate_patch(patch):
     except Exception as e:
         raise GenericConfigUpdaterError(f"Validate json patch: {patch} failed due to:{e}")
 
+
+def multiasic_validate_single_file(filename):
+    ns_list = [DEFAULT_NAMESPACE, *multi_asic.get_namespace_list()]
+    file_input = read_json_file(filename)
+    file_ns_list = [DEFAULT_NAMESPACE if key == HOST_NAMESPACE else key for key in file_input]
+    if set(ns_list) != set(file_ns_list):
+        diff = set(ns_list) - set(file_ns_list)
```

Comment:
>diff [](http://example.com/codeflow?start=8&length=4)

If the purpose is to print error message, just directly print them in original content, do not manipulate or combine.